.github/workflows/release_parallel.yml

```diff
@@ -111,85 +98,43 @@ jobs:
               continue-on-error: true
               run: npm test
 
-            - uses: ./.github/actions/configure_git
+            - name: Create branch name
+              id: create-branch-name
+              run: |
+                  SAFE_PATH=$(echo '${{ matrix.test.displayPath }}' | sed 's/\//-/g')
```

Comment:
Consider abstracting this out since it is used in at least 2 places and is a tad complicated